keras2onnx/ke2onnx/bidirectional.py

```diff
@@ -253,9 +253,12 @@ def convert_bidirectional(scope, operator, container):
             perm = [1, 0, 2] if container.target_opset <= 5 else [2, 0, 1, 3]
             apply_transpose(scope, lstm_h_name, transposed_h_name, container, perm=perm)
 
+            # Maintain backwards opset compatibility for 'Flatten'
+            target_opset = 1 if container.target_opset < 9 else 9
+
             # Flatten ONNX (N, D, C') into (N, D * C')
             container.add_node('Flatten', transposed_h_name, operator.outputs[0].full_name,
-                               name=scope.get_unique_variable_name('Flatten'), axis=1)
+                               name=scope.get_unique_variable_name('Flatten'), axis=1, op_version=target_opset)
```

Comment:
Semantically,  op_version here is op.since_version, not the parameter the user specified.

Reply:
addressed by renaming variable to op_version (as consistent with other conversions)